billing_inc.php

```diff
@@ -149,10 +145,22 @@
 		</tr>
 <?php
 	}
+?>
+<?php
+	if( $f_project_id != ALL_PROJECTS ) {
+?>
+		<tr class="row-1">
+			<td>
+				<input type="checkbox" name="subprojects" <?php if($f_subprojects) echo 'checked'; ?>>
```

Comment:
Here is an example of how we add the checked flag:

    <input type="checkbox" class="ace" id="bugnote_add_view_status" name="private" <?php check_checked( $t_default_bugnote_view_status, VS_PRIVATE ); ?> />